mstickerlib/src/matrix/sticker_formats/ponies.rs

```diff
@@ -35,6 +36,17 @@ pub struct MetaData {
 	pub size: usize,
 	pub mimetype: String
 }
+impl TryFrom<crate::image::Image> for MetaData {
```

Comment:
Why do we need a `TryFrom` implementation that can never fail?

Reply:
it can fail see
```rust
imetype: value.mime_type()?
```